RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -1067,15 +944,29 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       auto const gridThreadExtent = alpaka::getWorkDiv<alpaka::Grid, alpaka::Threads>(acc);
 
       for (uint16_t i = globalThreadIdx[0]; i < *modulesInGPU.nLowerModules; i += gridThreadExtent[0]) {
-        if (mdsInGPU.nMDs[i] == 0 or hitsInGPU.hitRanges[i * 2] == -1) {
+        if (mdsOccupancy.nMDs()[i] == 0 or hitsInGPU.hitRanges[i * 2] == -1) {
           rangesInGPU.mdRanges[i * 2] = -1;
           rangesInGPU.mdRanges[i * 2 + 1] = -1;
         } else {
           rangesInGPU.mdRanges[i * 2] = rangesInGPU.miniDoubletModuleIndices[i];
-          rangesInGPU.mdRanges[i * 2 + 1] = rangesInGPU.miniDoubletModuleIndices[i] + mdsInGPU.nMDs[i] - 1;
+          rangesInGPU.mdRanges[i * 2 + 1] = rangesInGPU.miniDoubletModuleIndices[i] + mdsOccupancy.nMDs()[i] - 1;
         }
       }
     }
   };
 }  // namespace ALPAKA_ACCELERATOR_NAMESPACE::lst
+
+namespace cms::alpakatools {
+  // This is not used, but it is needed for compilation
+  template <typename T0, typename... Args>
+  struct CopyToHost<PortableHostMultiCollection<T0, Args...>> {
```

Comment:
Could you elaborate? Having to copy a "host collection" to host seems wrong.

(same question as in https://github.com/SegmentLinking/cmssw/pull/93#discussion_r1784757797, assuming the answer is the same, one answer would be sufficient)

Reply:
Answered in #93. I should have put this in a common header since it is not specific to that class